app/modules/kit-library/assets/js/components/apply-kit-dialog.js

```diff
@@ -7,7 +7,8 @@ export default function ApplyKitDialog( props ) {
 
 	const startImportProcess = useCallback( ( applyAll = false ) => {
 		let url = '/import/process' +
-			`?file_url=${ encodeURIComponent( props.downloadLink ) }` +
+			`?id=${ props.id }` +
+			`&file_url=${ encodeURIComponent( props.downloadLink ) }` +
 			`&nonce=${ props.nonce }&referrer=kit-library`;
```

Comment:
At this stage of the concatenation, it might be the time to start using the `URL()` object IMO, it's too big and the next dev can easily break it

Reply:
This is a relative URL used after with the `navigate` not a full one, can't use the URL class here

Reply:
So you can use this:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams